pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+name = "treearbo"
+description = "Parser for .tree format"
+requires-python = ">=3.11"
```

Comment:
Why did you choose version 3.11 as the minimum? Have you tried running the project on version 3.9 (which is still officially supported)?

Reply:
because of the types like `from typing import Self, TypeAlias` and `TreePath: TypeAlias = str | int | None` they are 3.11+

Reply:
Thanks! I'll take it like it is. Later I'll make it work with 3.9+